### PR TITLE
Added variants property

### PR DESCRIPTION
**Pull request by wesleimp**

### CODE REVIEW

**Comment by ivarconr** [comment]
Nice! :fire:

File: src/Unleash/DefaultUnleash.cs [hunk 1]
```diff
@@ -88,6 +91,71 @@ private bool CheckIsEnabled(string toggleName, UnleashContext context, bool defa
...
+
+            if (position.HasValue)
+            {
+                return variants[position.Value];
+            }
+
+            return null;
+        }
+
+        private int? GetWeightedPosition(int[] weights)
```
Comment (silviogreuel) [inline src/Unleash/DefaultUnleash.cs:110]: In need of a better solution here, or this is ok?

**Comment by silviogreuel** [comment]
@stiano and @ivarconr, maybe, turn this repository as official sdk under unleash domain? to be more central and to gain some velocity on some replies/isues? tnks lads, great work you both...

**Comment by silviogreuel** [comment]
Hey @stiano @ivarconr, Just to notify, using this repository as base for https://github.com/Unleash/unleash-client-core

**Pull request closed without merging**